### PR TITLE
Add model_name_plural to ButtonHelper when initialising

### PR DESCRIPTION
**Pull request by Luperi**

### CODE REVIEW

File: wagtailmodeladmin/helpers.py [hunk 1]
```diff
@@ -189,6 +189,7 @@ def __init__(self, model, permission_helper, user,
         self.permission_helper = permission_helper
         self.inspect_view_enabled = inspect_view_enabled
         self.model_name = force_text(self.opts.verbose_name).lower()
+        self.model_name_plural = force_text(self.opts.verbose_name_plural).lower() if self.opts.verbose_name_plural else self.model_name
```
Comment (ababic) [inline wagtailmodeladmin/helpers.py]: @Luperi As far as I'm aware, `verbose_name_plural` is always populated in a model's `_meta`, whether defined in the model's `Meta` class or not. So, this should be enough:

`self.verbose_name_plural = force_text(self.opts.verbose_name_plural).lower()`

Comment (Luperi) [inline wagtailmodeladmin/helpers.py]: Ok good :)

**Comment by Luperi** [comment]
Are you going to merge this pull request soon? Thx

**Comment by ababic** [comment]
Hi @Luperi. I was waiting for you to update before I could merge, but looks like you've done that now. There's some general tidying I'd like to do on the treatment (forcing case changes) of `verbose_name` and `verbose_name_plural` throughout, which I hope to look at soon. This would go along nicely with that. I have a lot of client work on currently though, which needs to take precedence.

**Pull request closed without merging**